cmd/get-authkey/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// get-authkey allocates a one-time-use authkey using an OAuth API client
```

Comment:
Doc says one time use, but it also accepts reusable flag?

Reply:
Done